sql/src/main/java/io/crate/operation/reference/information/TablesSettingsExpression.java

```diff
@@ -75,6 +76,34 @@ public BytesRef value() {
         }
     }
 
+    static class TablesSettingsMappingExpression extends AbstractTablesSettingsExpression {
+
+        public static final String NAME = "mapping";
+
+        public TablesSettingsMappingExpression() {
+            addChildImplementations();
+        }
+
+        private void addChildImplementations() {
+            childImplementations.put(TablesSettingsMappingTotalFieldsExpression.NAME, new TablesSettingsMappingTotalFieldsExpression());
```

Comment:
I'd inline the `put` calls into the constructor - imho putting this in a function doesn't increase readability and afaik it disables some optimizations the JVM can do on object constructions.